services/jenkins/jenkins-plugin-installs.service.js

```diff
@@ -98,8 +98,7 @@ class JenkinsPluginInstalls extends BaseJsonService {
   static get route() {
     return {
       base: 'jenkins/plugin/i',
-      format: '([^/]+)/?([^/]+)?',
-      capture: ['plugin', 'version'],
+      pattern: ':plugin/:version?',
```

Comment:
Another tangential thought but could we update the second example title to include something like `version`?